src/test/cypress/fixtures/text_exercise_submission/text_exercise_submission.json

```diff
@@ -0,0 +1,3 @@
+{
+    "text": "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish."
+}
```

Comment:
This json only has one field for the text. I would suggest using a textfile as fixture here

Reply:
Generally we could but all other exercise submissions are in json format so i would prefer to keep the files uniform. Also i dont see what would be gained by having it be a .txt file instead.